com.github.ransome1.sleek.yml

```diff
@@ -0,0 +1,60 @@
+app-id: com.github.ransome1.sleek
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '20.08'
+separate-locales: false
+command: sleek
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --filesystem=home
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node14
+modules:
+  - name: sleek
+    build-options:
+      append-path: /usr/lib/sdk/node14/bin
+      env:
+        XDG_CACHE_HOME: /run/build/sleek/flatpak-node/cache
+        npm_config_nodedir: /usr/lib/sdk/node14
+    buildsystem: simple
+    build-commands:
+      - npm install --prefix=sleek --offline --cache=$FLATPAK_BUILDER_BUILDDIR/flatpak-node/npm-cache
+      - . flatpak-node/electron-builder-arch-args.sh; npm run --prefix=sleek --offline pack -- $ELECTRON_BUILDER_ARCH_ARGS
+      - mv sleek/dist/linux*unpacked /app/sleek
+      - install -Dm755 sleek.sh /app/bin/sleek
+    sources:
+      - type: git
+        url: https://github.com/ransome1/sleek.git
+        commit: 1bcdc188a9f47eb08116fe532bd8e2ba076b7b04
+        dest: sleek
+      - generated-sources.json
```

Comment:
FYI, this has been generated with `./flatpak-node-generator.py npm --xdg-layout package-lock.json`.